### PR TITLE
Added Express edit feature

### PR DESCRIPTION
**Pull request by Apochilles**
Please review my additions to Express edit method and routes.

### CODE REVIEW

File: client/src/components/beer-edit.jsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import React, { useCallback, useState } from "react";
```
Comment (NoxHarmonium) [inline client/src/components/beer-edit.jsx]: Please remove this file since it does nothing yet

File: server/src/index.js [hunk 2]
```diff
@@ -10,17 +10,38 @@ app.use(express.static("static"));
 app.use(bodyParser.json());
 
 app.get("/beers", async (req, res) => {
-  const beers = await beerService.getBeers();
-  res.send({
-    beers
-  });
+  // TODO: Use this try/catch pattern with the other handlers
```
Comment (NoxHarmonium) [inline server/src/index.js]: Please remove this comment

File: server/src/schemas/beer.js [hunk 3]
```diff
@@ -3,9 +3,19 @@ import mongoose from "mongoose";
...
-  rating: Number
+  brewery: {
+    type: String,
+    required: true
+  },
+  name: {
+    type: String,
+    required: true
+  },
+
```
Comment (NoxHarmonium) [inline server/src/schemas/beer.js]: Please remove this space to make the spacing consistent

**Review by NoxHarmonium — CHANGES_REQUESTED** [review]
Thanks. Some small changes needed.

File: client/src/components/beer-edit.jsx (hunk 1 above)
Comment (Apochilles) [reply client/src/components/beer-edit.jsx]: Thanks. I set it up to work on later but it's good to learn you should only commit actual features.

File: server/src/index.js (hunk 2 above)
Comment (Apochilles) [reply server/src/index.js]: Comment removed, will try to do this in future.

File: server/src/schemas/beer.js (hunk 3 above)
Comment (Apochilles) [reply server/src/schemas/beer.js]: Space removed. I just thought auto format would correct It, sorry.

**Review by NoxHarmonium — APPROVED** [review]
LGTM!

**Pull request merged**